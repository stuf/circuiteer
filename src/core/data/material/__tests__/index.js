import { natural, refined, composite, atmospheric } from '../index';

test('works', () => {
  expect({ natural, refined, composite, atmospheric }).toMatchSnapshot();
});
