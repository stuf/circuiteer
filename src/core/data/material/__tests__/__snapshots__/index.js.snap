// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
Object {
  "atmospheric": Object {
    "hydrogen": Object {
      "type": "atmospheric",
    },
  },
  "composite": Object {
    "plastic": Object {
      "from": Array [
        "compound",
        "coal",
      ],
      "type": "composite",
    },
    "rubber": Object {
      "from": Array [
        "organic",
        "resin",
      ],
      "type": "composite",
    },
  },
  "natural": Object {
    "compound": Object {
      "type": "natural",
    },
    "organic": Object {
      "type": "natural",
    },
    "resin": Object {
      "type": "natural",
    },
    "soil": Object {
      "type": "natural",
    },
  },
  "refined": Object {
    "coal": Object {
      "from": "organic",
      "type": "refined",
    },
  },
}
`;
